bundles/binding/org.openhab.binding.km200/src/main/java/org/openhab/binding/km200/internal/KM200SwitchProgramService.java

```diff
@@ -345,11 +321,24 @@ void updateSwitches(JSONObject nodeRoot) {
             /* Update the list of switching points */
             removeAllSwitches();
             JSONArray sPoints = nodeRoot.getJSONArray("switchPoints");
+            logger.debug("sPoints: {}", nodeRoot.toString());
```

Comment:
You never need to call `.toString()` on logger arguments and should not, as you can leave it up to the logger to call `.toString()` on any `java.lang.Object` only when needed.  Some `.toString()` methods are expensive, or at least create short-lived objects that add burden to the Java garbage collection.